src/Kodify/BlogBundle/Entity/AbstractBaseEntity.php

```diff
@@ -6,80 +6,5 @@
 
 abstract class AbstractBaseEntity
 {
-    /**
-     * @var \DateTime
-     *
-     * @ORM\Column(name="createdAt", type="datetime")
-     */
-    protected $createdAt;
 
-    /**
-     * @var \DateTime
-     *
-     * @ORM\Column(name="updatedAt", type="datetime")
-     */
-    protected $updatedAt;
-
-    /**
-     * Set createdAt
-     *
-     * @param \DateTime $createdAt
-     */
-    public function setCreatedAt($createdAt)
-    {
-        if (empty($this->createdAt)) {
-            $this->createdAt = $createdAt;
-        }
-
-        return $this;
-    }
-
-    /**
-     * Get createdAt
-     *
-     * @return \DateTime
-     */
-    public function getCreatedAt()
-    {
-        return $this->createdAt;
-    }
-
-    /**
-     * Set updatedAt
-     *
-     * @param \DateTime $updatedAt
-     */
-    public function setUpdatedAt($updatedAt = null)
-    {
-        $this->updatedAt = $updatedAt;
-
-        return $this;
-    }
-
-    /**
-     * @ORM\PreUpdate
-     * @ORM\PrePersist
-     */
-    public function markAsUpdated()
-    {
-        $this->setUpdatedAt(new \DateTime());
-    }
-
-    /**
-     * @ORM\PrePersist
-     */
-    public function initializeCreatedAt()
-    {
-        $this->setCreatedAt(new \DateTime());
-    }
-
-    /**
-     * Get updatedAt
-     *
-     * @return \DateTime
-     */
-    public function getUpdatedAt()
-    {
-        return $this->updatedAt;
-    }
```

Comment:
Hi Marcos,

Well... I had some problems testing with the database and with some perms
of files (there is a commit with a comment saying it :S) and it was
appearing some errors at createAt field, so I didn't want to spend so much
time solving silly errors and I deleted this fields... I know this is a
very poor solution...

2015-12-10 13:45 GMT+01:00 Marcos Estévez Martínez <notifications@github.com
>:

> In src/Kodify/BlogBundle/Entity/AbstractBaseEntity.php
> <https://github.com/natxo-kodify/blog/pull/12#discussion_r47221592>:
>
> > -     * @ORM\PrePersist
> > -     */
> > -    public function initializeCreatedAt()
> > -    {
> > -        $this->setCreatedAt(new \DateTime());
> > -    }
> > -
> > -    /**
> > -     * Get updatedAt
> > -     *
> > -     * @return \DateTime
> > -     */
> > -    public function getUpdatedAt()
> > -    {
> > -        return $this->updatedAt;
> > -    }
>
> Why did you delete all those lines?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/natxo-kodify/blog/pull/12/files#r47221592>.
>



-- 
Daniel
PHP & Symfony2 developer & Zitro